cobigen-eclipse/cobigen-eclipse-test/src/main/java/com/devonfw/cobigen/eclipse/test/common/utils/EclipseUtils.java

```diff
@@ -187,7 +187,7 @@ public static void cleanWorkspace(SWTWorkbenchBot bot, boolean cleanCobiGenConfi
         }
         break;
       } catch (Exception e) {
-        LOG.debug("An error occured while trying to delete project: {}", projectName);
+        LOG.debug("An error occured while trying to delete project: {} {}", projectName, e);
```

Comment:
No need to put another {} placeholder here. Root cause is added as last parameter always